entity-framework/core/miscellaneous/events/index.md

```diff
@@ -24,4 +39,46 @@ EF Core logs can be accessed from any type of application through use of [LogTo]
 
 This concept is similar to <xref:System.Data.Entity.Database.Log?displayProperty=nameWithType> in EF6.
 
-See [Simple Logging](xref:core/miscellaneous/events/simple-logging) for more information.
+See [Simple logging](xref:core/miscellaneous/events/simple-logging) for more information.
```

Comment:
Above: "EF Core logs can be accessed from any type of application through use of" should be "EF Core logs can be accessed from any type of application through *the* use of"